qemu/tests/cfg/qemu_cpu.cfg

```diff
@@ -18,19 +18,26 @@
     auto_cpu_model = no
 
     # QEMU versions:
-    variants qemu_flavor:
-        - @unknown:
-        - rhel:
-            variants rhel_major_version:
+    variants:
+        - @qemu_flavor_unknown:
+        - qemu_flavor_rhel:
+            variants:
                 - 6:
-                    variants rhel_version:
+                    only Host_RHEL.6
+                    variants:
                         - 6_0:
                         - 6_1:
                         - 6_2:
                         - 6_3:
                         - 6_4:
-        - upstream:
-            variants qemu_version:
+                - 7:
+                    only Host_RHEL.7
+                    variants:
```

Comment:
some scenerial tested on Host_RHEL.6 only or Host_RHEL.7 only, 
no diff between 6.xx , or 7.xx until you set it. didn't remove 6.xx or  as it's set by the owner, I just just update the format